platform/embedded-install.sh

```diff
@@ -100,7 +100,7 @@ if [ ! -z "$imagePullUsername" ] && [ ! "$imagePullPAT" = "null" ]; then
 fi
 
 pullSecretArgs2=()
-if [ ! "$imagePullPAT2" = "null" ]; then
+if [ ! -z "$imagePullPAT2" ]; then
```

Comment:
this just wasnt catching and it was adding a secondary pull secret where there was no first pull secret

Reply:
👍 thanks!